docs/release-notes/RELEASE-NOTES-0.0.0-iris.md

```diff
@@ -0,0 +1,79 @@
+# About this release
+
+[This](https://github.com/aeternity/aeternity/releases/tag/v0.0.0-iris) is a Iris test release.
+
+It:
```

Comment:
"It:"?  Maybe "Changes:"

Reply:
this is kind of template we use for long time

Reply:
Ok, I guess it's fine.